js-miniapp-sample/src/pages/landing.js

```diff
@@ -126,129 +131,153 @@ const Landing = (props: LandingProps) => {
       });
   }
 
+  function isLoggedIn() {
+    MiniApp.miniappUtils
+      .isLoggedIn()
+      .then((response) => {
+        setLoggedIn(response);
+      })
+      .catch((miniAppError) => {
+        console.log('getDarkMode - Error: ', miniAppError);
```

Comment:
I think this log should be 'isLoggedIn - error' please check.

Reply:
Done